source/docs/03-metadata.xml

```diff
@@ -1677,6 +1677,53 @@
+                  <p>The following example demonstrates how to structure detailed information about a repository (including the use of <gi scheme="MEI">identifier</gi>):</p>
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample088.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+                  <p>
+                     <specList>
+                        <specDesc key="treatHist"/>
+                     </specList>
+                  </p>
+                  <p>Conservation activities are often necessary to ensure the long-term preservation and integrity of manuscripts or printed sources. These conservation activities might include re-binding, restoration, modifying paper chemistry, and the like. In MEI the <gi scheme="MEI">treatHist</gi> element is intended to capture records of any treatment an item has undergone and may even specify details of the individual treatment process. <gi scheme="MEI">treatHist</gi> allows the option for either text or more structured information when formulating the specific encoding. It is recommended to make use of <gi scheme="MEI">p</gi> elements when a text-centred encoding is favoured and to use the <gi scheme="MEI">eventList</gi> element for a more structured encoding. Like <gi scheme="MEI">exhibHist</gi>, <gi scheme="MEI">treatHist</gi> is conceptually bound to the item-level. As an element it is thus not permitted at the work or expression level and only permitted at the manifestation level, if the manifestation comprises a manifestation singleton.</p>
+                  <p>
+                     <specList>
+                        <specDesc key="treatSched"/>
+                     </specList>
+                  </p>
+                  <p>Similar to the MEI <gi scheme="MEI">treatHist</gi> element, the <gi scheme="MEI">treatSched</gi> element is intended to hold records of conservation activities or treatments in regard to a bibliographic item. However, in contrast to <gi scheme="MEI">treatHist</gi>, <gi scheme="MEI">treatSched</gi> allows records of any anticipated activities, rather than simply a historical account of previous treatments. This might include any description indicating the quantity and frequency of the accruals. <gi scheme="MEI">treatSched</gi> furthermore may also be used to indicate that no additions or treatments are to be expected. To that end, <gi scheme="MEI">treatSched</gi> allows the option for either text or more structured information when formulating a specific encoding. It is at the discretion of the encoder to decide which specifics of the encoding are most appropriate.</p>
```

Comment:
```suggestion
                  <p>Similar to the MEI <gi scheme="MEI">treatHist</gi> element, the <gi scheme="MEI">treatSched</gi> element is intended to hold records of conservation activities or treatments in regard to a bibliographic item. However, in contrast to <gi scheme="MEI">treatHist</gi>, <gi scheme="MEI">treatSched</gi> allows records of any anticipated activities, rather than simply a historical account of previous treatments. This might include any description indicating the quantity and frequency of the treatments. <gi scheme="MEI">treatSched</gi> furthermore may also be used to indicate that no additions or treatments are to be expected. To that end, <gi scheme="MEI">treatSched</gi> allows the option for either text or more structured information when formulating a specific encoding. It is at the discretion of the encoder to decide which specifics of the encoding are most appropriate.</p>
```